pkg/policy/policy.go

```diff
@@ -33,7 +36,17 @@ type DefaultRuleDefinition struct {
 }
 
 func GetDefaultRules() (*DefaultRulesDefinitions, error) {
-	defaultRulesDefinitions, err := yamlToStruct(defaultRulesYamlContent)
+	configDefaultRulesYamlContent, err := getDefaultRulesFromFile()
+
+	var defaultRulesDefinitions *DefaultRulesDefinitions
+	if err == nil {
+		defaultRulesDefinitions, err = yamlToStruct(configDefaultRulesYamlContent)
```

Comment:
you can just do this in both `if` and `else`, might look cleaner
```return yamlToStruct(configDefaultRulesYamlContent)```

Reply:
Correct